em/emResidualFunctionals.cc

```diff
@@ -0,0 +1,932 @@
+/*
+ * Copyright (C) 2011 Scientific Computation Research Center
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+#include <iostream>
+#include <cstdlib>
+#include <algorithm>
+
+#include <apfCavityOp.h>
+#include "apfElement.h"
+#include "crv.h"
+#include "crvShape.h"
+
+#include "em.h"
+
+using namespace std;
```

Comment:
The same as the comment for the line including  the `iostrem`

Reply:
Removed.